lib/isodoc/function/to_word_html.rb

```diff
@@ -16,14 +17,15 @@ def note?
     end
 
     def init_file(filename, debug)
-      filename = filename.gsub(%r{\.[^/.]+$}, "")
+      filepath = Pathname.new(filename)
+      filename = filepath.sub_ext('').to_s
       dir = "#{filename}_files"
       unless debug
         Dir.mkdir(dir) unless File.exists?(dir)
         FileUtils.rm_rf "#{dir}/*"
       end
       @filename = filename
-      @localdir = %r{/}.match(filename) ? filename.sub(%r{/[^/]+$}, "/") : "./"
+      @localdir = filepath.parent.to_s + '/'
```

Comment:
It's better but `filepath.parent.to_s + '/'` is not good especially for Windows.

@CAMOBAP795 can you refactor the whole `init_file` method and its caller `convert_init` to use `Pathname`?

Reply:
> It's better but filepath.parent.to_s + '/' is not good especially for Windows.

It works on windows, ruby handle this

> @CAMOBAP795 can you refactor the whole init_file method and its caller convert_init to use Pathname?

Good idea and I have tried but this requires massive refactoring in many places, I would like to ask to do this in separate PR/commit

Reply:
OK, let's do in new PR. Thanks!